modules/apps/layout/layout-page-template-admin-web/src/main/java/com/liferay/layout/page/template/admin/web/internal/headless/delivery/dto/v1_0/structure/importer/ContainerLayoutStructureItemImporter.java

```diff
@@ -94,18 +99,118 @@ public LayoutStructureItem addLayoutStructureItem(
 				(Map<String, Object>)definitionMap.get("layout");
 
 			if (layout != null) {
-				containerLayoutStructureItem.setContainerType(
+				containerLayoutStructureItem.setAlign(
+					AlignConverter.convertToInternalValue(
+						(String)containerLayout.get("align")));
+				containerLayoutStructureItem.setBorderColor(
+					(String)containerLayout.get("borderColor"));
+				containerLayoutStructureItem.setBorderRadius(
+					BorderRadiusConverter.convertToInternalValue(
+						(String)containerLayout.get("borderRadius")));
+
+				Integer borderWidth = (Integer)containerLayout.get(
+					"borderWidth");
+
+				if (borderWidth != null) {
+					containerLayoutStructureItem.setBorderWidth(borderWidth);
+				}
+
+				String containerType = StringUtil.toLowerCase(
+					(String)containerLayout.get("containerType"));
+				String containerWidth = StringUtil.toLowerCase(
+					(String)containerLayout.get("containerType"));
+
+				if (containerWidth != null) {
```

Comment:
In here I am trying to "upgrade" the containerType, that now is named containerWidth